msteams-platform/includes/get-started/azure-provisioning-instructions-tab.md

```diff
@@ -22,6 +22,8 @@ Select the Teams Toolkit :::image type="icon" source="~/assets/images/teams-tool
 
     :::image type="content" source="~/assets/images/teams-toolkit-v2/deploy-azure/select-resource.png" alt-text="Screenshot showing resources for provisioning":::
 
+    For more information see [Create resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#create-resource-groups)
+
```

Comment:
Use the correct link (Link to Docs page by URL) tag in VSC. Check if you can add this information as part of the note below.